evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -449,36 +449,19 @@ def __init__(self, parameters, random_seed=0):
         )
 
 
-class SklearnStackedEnsemblePipeline(BinaryClassificationPipeline):
-    component_graph = ["Sklearn Stacked Ensemble Classifier"]
-
-    def __init__(self, parameters, random_seed=0):
-        super().__init__(
-            self.component_graph,
-            parameters=parameters,
-            random_seed=random_seed,
-        )
-
-
 pipelines_that_do_not_support_fast_permutation_importance = [
     PipelineWithDimReduction,
     PipelineWithDFS,
     PipelineWithCustomComponent,
     EnsembleDag,
-    SklearnStackedEnsemblePipeline,
```

Comment:
Our new impl also cannot be used for permutation importance right? `EnsembleDag` is that example?